turbinia/workers/volatility.py

```diff
@@ -59,23 +60,40 @@ def run(self, evidence, result):
             evidence.local_path, evidence.profile, self.module,
             output_file_path).split()
 
-    result.log('Running vol as [{0:s}]'.format(' '.join(cmd)))
-    res = self.execute(
-        cmd, result, log_files=[debug_log], new_evidence=[output_evidence],
-        close=True)
+    result.log('Running volatility as [{0:s}]'.format(' '.join(cmd)))
+    res = self.execute(cmd, result, new_evidence=[output_evidence], close=True)
 
     if res == 0:
-      # Write the report to the output file.
-      with open(output_file_path, 'w') as fh:
-        fh.write(output_evidence.text_data.encode('utf-8'))
-
-      # Add the resulting evidence to the result object.
-      result.add_evidence(output_evidence, evidence.config)
-      result.report_data = output_evidence
-      summary = 'Volatility module {0} successfully ran.'.format(self.module)
+      # Get report data from the output file.
+      try:
+        file_size = os.stat(output_file_path).st_size
+      except OSError as exception:
+        msg = 'Cannot read output file {0:s}: {1!s}'.format(
```

Comment:
'Cannot read output file' => 'Unable to determine size of output file'

Reply:
Done.